Zend/zend_execute_API.c

```diff
@@ -433,16 +433,20 @@ void shutdown_executor(void) /* {{{ */
 /* return class name and "::" or "". */
 ZEND_API const char *get_active_class_name(const char **space) /* {{{ */
 {
-	zend_function *func;
-
 	if (!zend_is_executing()) {
 		if (space) {
 			*space = "";
 		}
 		return "";
 	}
 
-	func = EG(current_execute_data)->func;
+	return get_class_name(EG(current_execute_data)->func, space);
+}
+/* }}} */
+
+/* return class name and "::" or "". */
+ZEND_API const char *get_class_name(const zend_function *func, const char **space) /* {{{ */
```

Comment:
Doesn't seem needed for anything?

Reply:
I thought that they will come handy in the future, but probably we shouldn't prematurely optimize for the future in this case :) so I've just removed the unneeded functions.